script.js

```diff
@@ -3,31 +3,54 @@ const charCount = document.getElementById("charCount");
 const submitBtn = document.getElementById("submitBtn");
 const maxChars = 250;
 
+
 tweetInput.addEventListener("input", () => {
   let inputLength = tweetInput.value.length;
 
+  if (inputLength === 0) {
+    submitBtn.classList.add("off");
+    submitBtn.disabled = true;
+  }
+   else 
+   {
+    submitBtn.classList.remove("off");
+    submitBtn.disabled = false;
+  }
+
```

Comment:
I see this logic is duplicated at line 38
let's extract it as function and call it in both case